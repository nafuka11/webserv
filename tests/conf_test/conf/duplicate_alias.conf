server {
    location / {
        alias ./docs;
        alias ./docs02;
    }
}