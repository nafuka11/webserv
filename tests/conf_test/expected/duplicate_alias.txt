webserv: duplicate "alias" directive in ./conf/duplicate_alias.conf:4
